vms/shift/templates/shift/edit_hours.html

```diff
@@ -1,6 +1,22 @@
 {% extends "vms/base.html" %}
 
 {% block content %}
+<script type="text/javascript">
```

Comment:
@tapasweni-pathak because some of the pages have no timepicker. For example, a page with a volunteer list has no timepicker. Or it is ok to have everywhere this code?